cwbi-auth-http-client/src/main/java/hec/army/usace/hec/cwbi/auth/http/client/trustmanagers/CwbiAuthTrustManager.java

```diff
@@ -0,0 +1,104 @@
+package hec.army.usace.hec.cwbi.auth.http.client.trustmanagers;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+import javax.security.auth.x500.X500Principal;
+
+public final class CwbiAuthTrustManager implements X509TrustManager {
+
+    private static final Logger LOGGER = Logger.getLogger(CwbiAuthTrustManager.class.getName());
+    private final TrustManagerFactory trustManagerFactory;
+
+    private static final X509TrustManager INSTANCE = buildTrustManager();
+
+    private CwbiAuthTrustManager(TrustManagerFactory trustManagerFactory) {
+        this.trustManagerFactory = trustManagerFactory;
+    }
+
+    /**
+     * Get X509TrustManager instance.
+     *
+     * @return Instance of X509TrustManager
+     */
+    private static X509TrustManager buildTrustManager() {
+        X509TrustManager retVal = null;
+        try (InputStream trustedCertificateAsInputStream = CwbiAuthTrustManager.class.getResourceAsStream("cumulusServer.pem")) {
+            KeyStore ts = KeyStore.getInstance("JKS");
+            ts.load(null, null);
+            Certificate trustedCertificate = CertificateFactory.getInstance("X.509").generateCertificate(trustedCertificateAsInputStream);
+            ts.setCertificateEntry("cwbi-auth-server-root-certificate", trustedCertificate);
+            TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance("PKIX");
+            trustManagerFactory.init(ts);
+            retVal =  new CwbiAuthTrustManager(trustManagerFactory);
+        } catch (CertificateException | NoSuchAlgorithmException | KeyStoreException | IOException e) {
+            String errorMsg = "Error retrieving X509TrustManager: " + e.getMessage();
```

Comment:
Pretty sure this is going to repeat e.getMessage as you included e in the log call